leapp/reporting/__init__.py

```diff
@@ -125,6 +128,30 @@ def __init__(self, value=None):
         self._value = value
 
 
+class Key(BasePrimitive):
+    """Stable identifier for report entries"""
```

Comment:
Regarding the guidelines, the DOT should be part of every docstring title.

```suggestion
    """Stable identifier for report entries."""
```